React/finger-exercises/react/ecmascript/02 - promises/index.js

```diff
@@ -1,9 +1,20 @@
 // Hint: use setInterval, create a new Promise and measure time with Date.now()
 
-export function delay() {
-
+export function delay(time) {
+  const startTime = Date.now();
+  return new Promise((resolve, reject) => {
+    setInterval(function() {
+      const endTime = Date.now();
+      const timeRandom = time - (endTime - startTime);
+      if ((timeRandom) > (time + 100)) {
```

Comment:
the parenthesis in `(timeRandom)` are unnecessary

Reply:
👍